python/ray/_private/parameter.py

```diff
@@ -130,6 +130,8 @@ class RayParams:
         session_name: The name of the session of the ray cluster.
         webui: The url of the UI.
         cluster_id: The cluster ID in hex string.
+        enable_physical_mode: Whether physical mode is enaled, which applies
```

Comment:
still typo and "memory" mentions needed here. Also, if a task does not specify memory do you still apply any constraints?

Reply:
Also I think this PR only adds parameter, so it's not really effective? If so, let's add it in the comment.

Reply:
Emm not sure if we need to mention `memory` here, since it's the validation phase, which should check against all resources, not only memory.

Reply:
Sure, I add some words into PR description; let me know if you think it's ok.